### PR TITLE
Added: Job

### PR DESCRIPTION
**Pull request by AB4B4B**
Reflecting comments in `Admiral Features` doc.

### CODE REVIEW

File: 0/README.md [hunk 1]
```diff
@@ -106,6 +106,10 @@ be assigned to individuals, enabling self-sovereign identity management
 
 See: https://w3c-ccg.github.io/did-spec/
 
+## Job
+
+A Job has a 1:1 mapping of an Orchestration. The result status of the Job can be used to inspect the status of execution for each step of the Orchestration (including potential failures in child Operations).
```
Comment (mikera) [inline 0/README.md]: Not technically 1:1 - an orchestration could be run multiple times each resulting in a separate job

File: 0/README.md [hunk 2]
```diff
@@ -106,6 +106,10 @@ be assigned to individuals, enabling self-sovereign identity management
...
+
```
Comment (mikera) [inline 0/README.md]: Worth adding a link to DEP6 where Jobs are defined.

**Comment by mikera** [comment]
@AB4B4B any updates? Or should we just close this?

File: 0/README.md (hunk 1 above)
Comment (AB4B4B) [reply 0/README.md]:
```suggestion
A Job allows a user to view an Orchestration. The result status of the Job can be used to inspect the status of execution for each step of the Orchestration (including potential failures in child Operations).
For further details, see [DEP6](https://github.com/DEX-Company/DEPs/tree/master/6).
```

File: 0/README.md (hunk 2 above)
Comment (AB4B4B) [reply 0/README.md]: link added

**Comment by mikera** [comment]
Looks good. Suggest you update and merge. You many need to do a "resolve conflicts" first

**Pull request merged**